sensor_msgs/Image image
---
std_msgs/Header header
sensor_msgs/Image image
Object[] objects
